data/entities.yaml

```diff
@@ -66,11 +66,19 @@
 - name: mountain
   display:
     attr: snow
-    char: '∧'
+    char: '▲'
```

Comment:
I am not convinced whether these characters will be supported in all terminal fonts.  Perhaps we should come up with an official policy re: what subset of Unicode the game uses, and perhaps pair that with some recommendations of what terminal font(s) will work well.

Reply:
Good point, the current chat is a bit confusing with robots, so I would personally prefer pointing people in the direction of better fonts. Though we could add a fallback `--ascii` option for DOS users that would pick `ascii: ^` in the YAML.

Reply:
Yeah, I originally had filled triangle chars for robots, but then I saw someone try it and those chars were double wide in their font =(

Reply:
I do not have the energy and time to do it right, so I found the characters `A` and `Å` instead. :mountain: